code/modules/mob/living/carbon/human/human_damage.dm

```diff
@@ -222,6 +222,15 @@
 			amount *= 0.5
 		..(amount)
 
+								//// TOXIN ORGAN ROT ////
+		if (ishuman(src))
+			if ((src.getToxLoss() > 75) && (amount>0)) // If toxloss is above a certain threshhold, more toxin damage will cause internal organ damage. For reference: 50 is DANGEROUS TOXIN LEVELS DETECTED
+				var/obj/item/organ/internal/targeted_organ
+				var/list/listed_organs  = list("brain",OP_EYES,"heart","lungs","stomach","liver","kidneys","appendix","psionic organ")
```

Comment:
What happends if theirs missing one of these?

Reply:
It tumbles out of the proc and doesn't do anything. This was put in at the very end of the toxic damage proc.